src/components/App.css

```diff
@@ -0,0 +1,55 @@
+body {
+  background-color: white;
+}
+
+.App {
+  display: flex;
+  flex-direction: column;
+  width: 700px;
+}
+
+.Display {
+  display: flex;
+  background-color: gray;
+  height: 100px;
+  color: white;
+  font-weight: bold;
+  font-size: 20px;
+  justify-content: right;
```

Comment:
Let's make our app looks like the original.
`0` should be at the right 

Yours;
![image](https://user-images.githubusercontent.com/31897434/95397821-18747c00-08fc-11eb-9b12-66492ebc6ee6.png)

Original;
![image](https://user-images.githubusercontent.com/31897434/95397885-4659c080-08fc-11eb-9643-c602a6f12365.png)


Try to use `flex-end`

```
justify-content: flex-end;
padding-right: 5px
```